src/main/scala/tech/cryptonomic/conseil/util/ConfigUtil.scala

```diff
@@ -0,0 +1,61 @@
+package tech.cryptonomic.conseil.util
+
+import com.typesafe.config.ConfigFactory
+import scala.collection.JavaConverters._
+
+import java.util.HashMap
+
+object ConfigUtil {
+
+  private val conf = ConfigFactory.load
+
+  /**
+    * Output Map, keys being blockchain platforms, values being comma and space
+    * separated string of their networks
+    * @return platform, network Map
+    */
+  def parsePlatformsAndNetworks(): Map[String, String] = {
+    val obj = conf.getObject("platforms").asScala.toMap
+    val platformsAndNetworks = obj.map { pair =>
+      val platforms = pair._1
+      val networks = pair._2
+        //convert map to network strings
+        .unwrapped().asInstanceOf[HashMap[String, String]].asScala.keySet
+        //collect using concatenation with commas
+        .foldLeft("")((acc, network) => acc + network + ", ")
+        //drop the last comma and space
+        .dropRight(2)
+      (platforms, networks)
+    }
+    platformsAndNetworks
+  }
+
+  /**
+    * Given a specific platform, output comma and space separated string of its
+    * networks
+    * @param platform Blockchain platform
+    * @return
+    */
+  def parseNetworksForAGivenPlatform(platform: String): String = {
+    val platformsAndNetworks = parsePlatformsAndNetworks()
+    platformsAndNetworks(platform)
```

Comment:
Consider this: 
using this form instead of `platformAndNetworks.get(platform)` will throw an `NoSuchElementException` if the input is not in the keyset.

the `get(...)` form returns an `Option` instead